.pre-commit-config.yaml

```diff
@@ -26,9 +26,9 @@ repos:
 
       - id: ruff-format
 
-  - repo: https://github.com/numpy/numpydoc
-    rev: v1.8.0
-    hooks:
-      - id: numpydoc-validation
-        files: ^src/
-        exclude: ^tests/
+  # - repo: https://github.com/numpy/numpydoc
```

Comment:
Let's comment this out for now so we're not blocking others, and we can turn it back on as we work on this?